Model/ShimModel.php

```diff
@@ -29,8 +29,13 @@ public function __construct($id = false, $table = null, $ds = null) {
 			$this->prefixOrderProperty();
 		}
 
-		$event = new CakeEvent('Model.initialize', $this, compact('id', 'table', 'ds'));
```

Comment:
If you want to be close to 3.x, then removing this doesnt sound like a good idea
see https://github.com/cakephp/cakephp/blob/master/src/ORM/Table.php#L293

Reply:
This event produced the error I posted above.

Reply:
What exactly did you use? The one so far worked.

Reply:
https://travis-ci.org/dereuromark/cakephp-shim/jobs/366295370

Reply:
ok